mozregression/download_manager.py

```diff
@@ -266,7 +266,7 @@ def wait(self, raise_if_error=True):
         """
         Wait for all downloads to be finished.
         """
-        downloads = self._downloads.values()
+        downloads = list(self._downloads.values())
         for download in downloads:
             download.wait(raise_if_error=raise_if_error)
```

Comment:
I think iterating over the keys is probably a little safer.

https://stackoverflow.com/a/45277800

```suggestion
        download_keys = 
        for download_keys in list(self._downloads.keys()):
            download = self._downloads.get(download_key)
            if download:
                download.wait(raise_if_error=raise_if_error)
```

Reply:
Earlier suggestion was actually wrong! Fixed

Reply:
Done, thanks!
I also added a little comment about the reason behind the `list()` call for future reference.